hack/dev-config.yaml

```diff
@@ -0,0 +1,573 @@
+apiVersion: v1
```

Comment:
Can we please auto-generate most of this and not duplicate the content? See the `hack/build-single-manifests.sh` script for reference. Most of it can be auto-gen except the ingress-kong deployment.